odd-platform-api/src/main/java/com/provectus/oddplatform/controller/TagController.java

```diff
@@ -46,7 +52,9 @@ public TagController(final TagService entityService) {
         @Valid final String query,
         final ServerWebExchange exchange
     ) {
-        return list(page, size, query);
+        return tagService.listMostPopular(query, page, size)
```

Comment:
tagService is accessible via this.entityService

Reply:
Oof, okay